<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.1.2 (Python 3.7.6 on win32)" generated="20200330 15:05:24.276" rpa="false">
<suite id="s1" name="TC Robot" source="C:\Users\admin\PycharmProjects\HTTPRequest\VideoGameAPI\TC_Robot.robot">
<test id="s1-t1" name="TC1:Return all videos games(GET)">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>mysession</arg>
<arg>${base_url}</arg>
</arguments>
<msg timestamp="20200330 15:05:25.353" level="INFO">Creating Session using : alias=mysession, url=http://localhost:8080, headers={},                     cookies={}, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
<status status="PASS" starttime="20200330 15:05:25.352" endtime="20200330 15:05:25.353"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>Accept=application/json</arg>
</arguments>
<assign>
<var>${header}</var>
</assign>
<msg timestamp="20200330 15:05:25.354" level="INFO">${header} = {'Accept': 'application/json'}</msg>
<status status="PASS" starttime="20200330 15:05:25.354" endtime="20200330 15:05:25.354"></status>
</kw>
<kw name="Get Request" library="RequestsLibrary">
<doc>Send a GET request on the session object found using the
given `alias`</doc>
<arguments>
<arg>mysession</arg>
<arg>/app/videogames</arg>
<arg>headers=${header}</arg>
</arguments>
<assign>
<var>${response}</var>
</assign>
<msg timestamp="20200330 15:05:25.355" level="INFO">GET Request using : uri=/app/videogames, params=None, allow_redirects=True, timeout=None
headers={'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection': 'keep-alive'} 
data=None 
json=None</msg>
<msg timestamp="20200330 15:05:29.414" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D189B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:33.625" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D23070&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:38.035" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D23250&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:42.114" level="FAIL">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03DBC1B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</msg>
<status status="FAIL" starttime="20200330 15:05:25.355" endtime="20200330 15:05:42.114"></status>
</kw>
<status status="FAIL" starttime="20200330 15:05:25.352" endtime="20200330 15:05:42.116" critical="yes">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03DBC1B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</status>
</test>
<test id="s1-t2" name="TC2:Add a new video game (POST)">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>mysession</arg>
<arg>${base_url}</arg>
</arguments>
<msg timestamp="20200330 15:05:42.123" level="INFO">Creating Session using : alias=mysession, url=http://localhost:8080, headers={},                     cookies={}, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
<status status="PASS" starttime="20200330 15:05:42.122" endtime="20200330 15:05:42.123"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>id=22</arg>
<arg>name=string22</arg>
<arg>releaseDate=2020-03-18T09:51:10.626Z</arg>
<arg>reviewScore=0</arg>
<arg>category=string</arg>
<arg>rating=string</arg>
</arguments>
<assign>
<var>${body}</var>
</assign>
<msg timestamp="20200330 15:05:42.128" level="INFO">${body} = {'id': '22', 'name': 'string22', 'releaseDate': '2020-03-18T09:51:10.626Z', 'reviewScore': '0', 'category': 'string', 'rating': 'string'}</msg>
<status status="PASS" starttime="20200330 15:05:42.126" endtime="20200330 15:05:42.128"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>Content-Type=application/json</arg>
</arguments>
<assign>
<var>${header}</var>
</assign>
<msg timestamp="20200330 15:05:42.130" level="INFO">${header} = {'Content-Type': 'application/json'}</msg>
<status status="PASS" starttime="20200330 15:05:42.129" endtime="20200330 15:05:42.130"></status>
</kw>
<kw name="Post Request" library="RequestsLibrary">
<doc>Send a POST request on the session object found using the
given `alias`</doc>
<arguments>
<arg>mysession</arg>
<arg>/app/videogames</arg>
<arg>data=${body}</arg>
<arg>headers=${header}</arg>
</arguments>
<assign>
<var>${response}</var>
</assign>
<msg timestamp="20200330 15:05:42.133" level="INFO">POST Request using : uri=/app/videogames, params=None, files=None, allow_redirects=True, timeout=None
headers={'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Type': 'application/json'} 
data={"id": "22", "name": "string22", "releaseDate": "2020-03-18T09:51:10.626Z", "reviewScore": "0", "category": "string", "rating": "string"} 
json=None</msg>
<msg timestamp="20200330 15:05:46.146" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A7B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:50.355" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A490&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:54.762" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A910&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:58.774" level="FAIL">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0AA10&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</msg>
<status status="FAIL" starttime="20200330 15:05:42.131" endtime="20200330 15:05:58.775"></status>
</kw>
<status status="FAIL" starttime="20200330 15:05:42.120" endtime="20200330 15:05:58.776" critical="yes">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0AA10&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</status>
</test>
<test id="s1-t3" name="TC3: Return the details of a single game by ID(GET)">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>mysession</arg>
<arg>${base_url}</arg>
</arguments>
<msg timestamp="20200330 15:05:58.782" level="INFO">Creating Session using : alias=mysession, url=http://localhost:8080, headers={},                     cookies={}, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
<status status="PASS" starttime="20200330 15:05:58.781" endtime="20200330 15:05:58.782"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>Accept=application/json</arg>
</arguments>
<assign>
<var>${header}</var>
</assign>
<msg timestamp="20200330 15:05:58.784" level="INFO">${header} = {'Accept': 'application/json'}</msg>
<status status="PASS" starttime="20200330 15:05:58.783" endtime="20200330 15:05:58.784"></status>
</kw>
<kw name="Get Request" library="RequestsLibrary">
<doc>Send a GET request on the session object found using the
given `alias`</doc>
<arguments>
<arg>mysession</arg>
<arg>/app/videogames/3</arg>
<arg>headers=${header}</arg>
</arguments>
<assign>
<var>${response}</var>
</assign>
<msg timestamp="20200330 15:05:58.785" level="INFO">GET Request using : uri=/app/videogames/3, params=None, allow_redirects=True, timeout=None
headers={'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection': 'keep-alive'} 
data=None 
json=None</msg>
<msg timestamp="20200330 15:06:02.795" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E250B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:07.002" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E25290&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:11.412" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E253F0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:15.421" level="FAIL">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/3 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E255B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</msg>
<status status="FAIL" starttime="20200330 15:05:58.784" endtime="20200330 15:06:15.422"></status>
</kw>
<status status="FAIL" starttime="20200330 15:05:58.780" endtime="20200330 15:06:15.422" critical="yes">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/3 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E255B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</status>
</test>
<test id="s1-t4" name="TC4:Update an existing video game by specifying new body (PUT)">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>mysession</arg>
<arg>${base_url}</arg>
</arguments>
<msg timestamp="20200330 15:06:15.424" level="INFO">Creating Session using : alias=mysession, url=http://localhost:8080, headers={},                     cookies={}, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
<status status="PASS" starttime="20200330 15:06:15.424" endtime="20200330 15:06:15.425"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>id=4</arg>
<arg>name=LongHo44</arg>
<arg>releaseDate=2020-03-18T09:51:10.626Z</arg>
<arg>reviewScore=0</arg>
<arg>category=longho</arg>
<arg>rating=string</arg>
</arguments>
<assign>
<var>${body}</var>
</assign>
<msg timestamp="20200330 15:06:15.425" level="INFO">${body} = {'id': '4', 'name': 'LongHo44', 'releaseDate': '2020-03-18T09:51:10.626Z', 'reviewScore': '0', 'category': 'longho', 'rating': 'string'}</msg>
<status status="PASS" starttime="20200330 15:06:15.425" endtime="20200330 15:06:15.425"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>Content-Type=application/json</arg>
<arg>Accept=application/json</arg>
</arguments>
<assign>
<var>${header}</var>
</assign>
<msg timestamp="20200330 15:06:15.426" level="INFO">${header} = {'Content-Type': 'application/json', 'Accept': 'application/json'}</msg>
<status status="PASS" starttime="20200330 15:06:15.425" endtime="20200330 15:06:15.426"></status>
</kw>
<kw name="Put Request" library="RequestsLibrary">
<doc>Send a PUT request on the session object found using the
given `alias`</doc>
<arguments>
<arg>mysession</arg>
<arg>/app/videogames/4</arg>
<arg>data=${body}</arg>
<arg>headers=${header}</arg>
</arguments>
<assign>
<var>${response}</var>
</assign>
<msg timestamp="20200330 15:06:15.426" level="INFO">PUT Request using : uri=/app/videogames/4, params=None, files=None, allow_redirects=True, timeout=None
headers={'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection': 'keep-alive', 'Content-Type': 'application/json'} 
data={"id": "4", "name": "LongHo44", "releaseDate": "2020-03-18T09:51:10.626Z", "reviewScore": "0", "category": "longho", "rating": "string"} 
json=None</msg>
<msg timestamp="20200330 15:06:19.434" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30370&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:23.640" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30550&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:28.050" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30690&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:32.061" level="FAIL">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/4 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03DBCD50&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</msg>
<status status="FAIL" starttime="20200330 15:06:15.426" endtime="20200330 15:06:32.062"></status>
</kw>
<status status="FAIL" starttime="20200330 15:06:15.423" endtime="20200330 15:06:32.062" critical="yes">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/4 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03DBCD50&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</status>
</test>
<test id="s1-t5" name="TC5: Deletes a video game by ID (DELETE)">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>mysession</arg>
<arg>${base_url}</arg>
</arguments>
<msg timestamp="20200330 15:06:32.064" level="INFO">Creating Session using : alias=mysession, url=http://localhost:8080, headers={},                     cookies={}, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
<status status="PASS" starttime="20200330 15:06:32.064" endtime="20200330 15:06:32.065"></status>
</kw>
<kw name="Create Dictionary" library="BuiltIn">
<doc>Creates and returns a dictionary based on the given ``items``.</doc>
<arguments>
<arg>Accept=application.json</arg>
</arguments>
<assign>
<var>${header}</var>
</assign>
<msg timestamp="20200330 15:06:32.065" level="INFO">${header} = {'Accept': 'application.json'}</msg>
<status status="PASS" starttime="20200330 15:06:32.065" endtime="20200330 15:06:32.065"></status>
</kw>
<kw name="Delete Request" library="RequestsLibrary">
<doc>Send a DELETE request on the session object found using the
given `alias`</doc>
<arguments>
<arg>mysession</arg>
<arg>/app/videogames/5</arg>
</arguments>
<assign>
<var>${response}</var>
</assign>
<msg timestamp="20200330 15:06:32.066" level="INFO">DELETE Request using : uri=/app/videogames/5, params=None, allow_redirects=True, timeout=None
headers={'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive'} 
data=None 
json=None</msg>
<msg timestamp="20200330 15:06:36.079" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E25B50&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
<msg timestamp="20200330 15:06:40.286" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E307B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
<msg timestamp="20200330 15:06:44.698" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E304F0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
<msg timestamp="20200330 15:06:48.713" level="FAIL">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/5 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30550&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</msg>
<status status="FAIL" starttime="20200330 15:06:32.066" endtime="20200330 15:06:48.713"></status>
</kw>
<status status="FAIL" starttime="20200330 15:06:32.063" endtime="20200330 15:06:48.715" critical="yes">ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /app/videogames/5 (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30550&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))</status>
</test>
<status status="FAIL" starttime="20200330 15:05:24.277" endtime="20200330 15:06:48.721"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="5">Critical Tests</stat>
<stat pass="0" fail="5">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="5" id="s1" name="TC Robot">TC Robot</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20200330 15:05:29.414" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D189B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:33.625" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D23070&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:38.035" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x00D23250&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:46.146" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A7B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:50.355" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A490&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:05:54.762" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E0A910&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames</msg>
<msg timestamp="20200330 15:06:02.795" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E250B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:07.002" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E25290&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:11.412" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E253F0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/3</msg>
<msg timestamp="20200330 15:06:19.434" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30370&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:23.640" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30550&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:28.050" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E30690&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/4</msg>
<msg timestamp="20200330 15:06:36.079" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E25B50&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
<msg timestamp="20200330 15:06:40.286" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E307B0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
<msg timestamp="20200330 15:06:44.698" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x03E304F0&gt;: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /app/videogames/5</msg>
</errors>
</robot>
